src/main/java/core/basesyntax/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> data) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        for (String line : data) {
```

Comment:
Don't begin method implementation with an empty line. According to the checklist, there should be no redundant empty lines at the start of class or method implementations.